_health-care/_js/components/form-elements/GrowableTable.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+
+class GrowableTable extends React.Component {
+  constructor(props) {
```

Comment:
This is apparently what we're supposed to have been doing? I think?  @crwallace did it in her commit that she sent for early review yesterday and I noticed it matched the example snippets.